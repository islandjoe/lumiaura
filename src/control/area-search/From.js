import React from 'react'
import '../../layout/AreaSearch.css'

const fromLoc =(props)=> {
  return (
    <div className="ui input">
      <input type="text" placeholder="From..." />
    </div>
  )
}

export default fromLoc