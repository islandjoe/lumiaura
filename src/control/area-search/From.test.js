import React from 'react'
import renderer from 'react-test-renderer'
import From from './From'

describe('<From/>', ()=> {
  test('should render', ()=> {
    const component = renderer
      .create(<From/>)
      .toJSON()

    expect(component).toMatchSnapshot()
  })
})