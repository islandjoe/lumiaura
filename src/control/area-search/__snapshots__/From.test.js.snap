// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<From/> should render 1`] = `
<div
  className="ui input"
>
  <input
    placeholder="From..."
    type="text"
  />
</div>
`;
